src/api/app/views/webui/request/show.html.erb

```diff
@@ -139,27 +139,14 @@
                 <% end %>
                 <%= project_or_package_link project: action[:tprj], package: action[:tpkg] %>
               </b>
-          <% elsif action[:type] == :add_role %>
-              <b>
-                <%= user_with_realname_and_icon action[:user] %>
-                wants the role <b><%= action[:role] %></b> for
-                <%= project_or_package_link project: action[:tprj], package: action[:tpkg] %>
-              </b>
-          <% elsif action[:type] == :set_bugowner %>
+          <% elsif [:add_role, :set_bugowner].include? action[:type]  %>
               <b>
                 <%= user_with_realname_and_icon @bs_request.creator %>
-                <% if @bs_request.creator == action[:user] %>
-                  wants to
-                <% else %>
-                  wants that the
-                  <% if action[:user] %>
-                    user
-                    <%= user_with_realname_and_icon(action[:user], { no_icon: true }) %>
-                  <% else %>
-                    "group #{action[:group]}"
-                  <% end %>
-                <% end %>
-                become bugowner for
+                wants
+                <%= requester_str(@bs_request.creator, action[:user], action[:group]).html_safe %>
```

Comment:
I guess the html safe is necessary because of the ``user_with_realname_and_icon(requester_user, no_icon: true)``? Then I would add the html_safe only to this string!